CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 * Fixed a crash when dismissing `NavigationViewController`. ([#4029](https://github.com/mapbox/mapbox-navigation-ios/pull/4029))
 * Fixed an issue where `StyleManager` was not applying style on iPad. ([#4039](https://github.com/mapbox/mapbox-navigation-ios/pull/4039))
 * Added filling of `Incident` properties `countryCodeAlpha3`, `countryCode`, `roadIsClosed`, `longDescription`, `numberOfBlockedLanes`, `congestionLevel`, `affectedRoadNames` when receiving `RoadObject` via Electronic Horizon. ([#4045](https://github.com/mapbox/mapbox-navigation-ios/pull/4045))
+* Fixes bearing calculation error during tunnel dead reckoning. ()
```

Comment:
If I'm not mistaken, Navigation Native entries that we'd like to mention in Navigation SDK should go to `Other changes` section without any linked PRs, so:
```suggestion
* Fixes bearing calculation error during tunnel dead reckoning.
```